src/tools/cephfs/top/cephfs-top

```diff
@@ -1138,7 +1154,13 @@ class FSTop(FSTopBase):
                     else:
                         num_client = len(client_metadata)
                     vscrollEnd += num_client
-                    if self.create_header(stats_json, help, screen_title, 2):
+                    try:
+                        header_create = self.create_header(stats_json, help, screen_title, 2)
+                    except curses.error:
+                        curses.endwin()
+                        sys.stderr.write('Error creating header\n')
+                        exit()
+                    if header_create:
```

Comment:
How about creating a helper func for this? `handle_header()`?

Reply:
fixed, thanks